client/gutenberg/extensions/presets/jetpack/editor-shared/get-site-fragment.js

```diff
@@ -26,5 +26,14 @@ export default function getSiteFragment() {
 		return siteFragment || null;
 	}
 
+	// Gutenberg in Jetpack adds a site fragment in the initial state
+	if (
```

Comment:
Given that wp-admin is likely to be our primary target, should we move this condition to the top?

Reply:
I'll leave it where it is to avoid the possibility of conflicting with the other conditionals, particularly the second conditional.